templates/basecurriculum.html

```diff
@@ -82,6 +82,7 @@
     <div class="header printbackground">
         <div class="logo">
             <img src="{% static "images/logo.png" %}">
+            <img src='{{ request.path | tracking_pixel_url}}'>
```

Comment:
What does this do exactly?

Reply:
It's Django shenanigans. 

In the other file, we wrote a custom template filter,
 `tracking_pixel_url`, which takes one parameter, `path`.  This line is how we call that function with `request.path` as an argument. The return value of which, the tracking pixel url that hits studio.code.org, is then formatted as a string and used as the source for the image.

https://docs.djangoproject.com/en/2.2/howto/custom-template-tags/